.teamcity/settings/GradleProfilerPublishing.kt

```diff
@@ -34,7 +37,7 @@ object GradleProfilerPublishing : BuildType({
 
     steps {
         gradle {
-            tasks = "clean publishToSonatype closeAndReleaseSonatypeStagingRepository gitPushTag releaseToSdkMan %additional.gradle.parameters%"
+            tasks = "clean createBuildReceipt publishToSonatype closeAndReleaseSonatypeStagingRepository gitPushTag releaseToSdkMan %additional.gradle.parameters%"
```

Comment:
Here we still have releaseToSdkMan, is that ok?

Reply:
No, it is not. I removed it.